pages/mi/mi-github-issues.md

```diff
@@ -164,14 +164,21 @@ Raw.githack makes it easy to see what the page will look like if your branch is
 
 Now, go to your forked repository on GitHub by visiting `https://github.com/YourUsername/YourUsername.github.io`. GitHub will detect that you recently pushed a new branch.
 
-1. Click the blue "Compare & pull request" button.
+1. Click the "Compare & pull request" button.
+  - If it doesn't appear, go to the "Pull requests" tab and click the "New pull request" button. In the "compare:master" dropdown, choose the branch you were working on.
 
-   ![Compare & Pull Button](image/mi-compare-pull.png)
+   ![Compare & Pull Button](image/mi-compare-pull-updated.png)
 
 2. Follow the pull request template carefully and complete the necessary details in the description.
 
    ![Pull Request Template](image//mi-pr-title-and-description.png)
 
+   Things to check before creating PR:
+      - add title and issue number in title
+      - fill the checkboxes
```

Comment:
We used to have a similar list, and I believed it was causing prospective virtual interns to overlook the pull request template, possibly because of an outdated screenshot of the template. Therefore, I updated the PR template, removed the list entirely and replaced it with: "Follow the pull request template carefully and complete the necessary details in the description."

Perhaps we could just bold the text `Follow the pull request template carefully`?

If you have feedback about the pull request template, please let us know as well.

Reply:
Yes making it bold would definitely grab attention. I really liked the PR template, especially tagging the issue number and linking the RawLink. We can see how it actually looks before deploying it to the main website and request changes if something's off.

Reply:
Got you. I see your list is meant to provide an overview the pr template. Please go ahead and bold the text as we discussed.